### PR TITLE
Update example READMEs

### LINKED ISSUE #676: Example READMEs are out of date
### Environment
Device and OS: System76 darp8, popOS
App version: v0.21.2
Kubernetes distro being used: N/A
Other: N/A

### Steps to reproduce
1. Navigate to the `examples/` sub directory in the repo

### Expected result
The examples list is complete and reflects valid information.

### Actual Result
The examples list is incomplete and reflects invalid information (i.e. the BB demo is not on v1.33).

### PR DESCRIPTION
**Pull request by Racer159**
## Description

This PR updates the examples to be simpler and to accurately reflect the current state of Zarf.  Will add a PR to the `zarf-docs` repo to pull these in as well.

## Related Issue

Fixes #676 

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)

## Checklist before merging

- [X] Documentation has been updated as necessary (add the `needs-docs` label)
- [X] (Optional) Changes have been linted locally with [golangci-lint](https://github.com/golangci/golangci-lint). (NOTE: We haven't turned on lint checks in the pipeline yet so linting may be hard if it shows a lot of lint errors in places that weren't touched by changes. Thus, linting is optional right now.)

### CODE REVIEW

File: examples/postgres-operator/README.md [hunk 1]
```diff
@@ -14,18 +24,22 @@ After looking at several alternatives, Zalando's postgres operator felt like the
 
 1. Clone the Zarf project &mdash; for the example configuration files.
 
-1. Download a Zarf release &mdash; you need a binary _**and**_ an init package, [here](../../docs/workstation.md#just-gimmie-zarf).
+1. Log into [Iron Bank](https://registry1.dso.mil) if you haven't already. (Required for the `minio` images in this example)
```
Comment (jeff-mccoy) [inline examples/postgres-operator/README.md:27]: can you remove this, IB images are no longer required for the postgres/minio examples

File: docs/13-walkthroughs/3-add-logging.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+In this walkthrough, we are going to show how you can use a Zarf component to inject zero-config, centralized logging into your Zarf cluster.
+
+More specifically, you'll be adding a [Promtail / Loki / Grafana (PLG)](https://github.com/grafana/loki) stack to the [Doom Walkthrough](./2-deploying-doom.md) by installing Zarf's "logging" component.
+
+
+## Walkthrough Prerequisites
+1. The [Zarf](https://github.com/defenseunicorns/zarf) repository cloned: ([`git clone` Instructions](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository))
+1. Zarf binary installed on your $PATH: ([Install Instructions](../3-getting-started.md#installing-zarf))
+1. kubectl: ([kubectl Install Instructions](https://kubernetes.io/docs/tasks/tools/#kubectl))
```
Comment (jeff-mccoy) [inline docs/13-walkthroughs/3-add-logging.md:11]: Think this is old, we shouldn't need kubectl I don't think...

**Review by jeff-mccoy — COMMENTED** [review]
just a couple comments, overall fantastic / detailed work.

File: docs/13-walkthroughs/3-add-logging.md (hunk 2 above)
Comment (Racer159) [reply docs/13-walkthroughs/3-add-logging.md]: Removed, it is mentioned as a way to validate your cluster on the Zarf init walkthrough, but added a note to that to say it is optional.

File: examples/postgres-operator/README.md (hunk 1 above)
Comment (Racer159) [reply examples/postgres-operator/README.md]: Changing things out from underneath me I see :sweat_smile:

File: docs/13-walkthroughs/3-add-logging.md (hunk 2 above)
Comment (jeff-mccoy) [reply docs/13-walkthroughs/3-add-logging.md]: Yeah that was from before we did wait logic for cluster readiness or had k9s vendored in.

Comment (jeff-mccoy) [reply docs/13-walkthroughs/3-add-logging.md]: Actually we should change this to "zarf tools monitor".  For our primary personas kubectl will have no advantage for k9s and in fact will be harder for many.

Comment (Racer159) [reply docs/13-walkthroughs/3-add-logging.md]: yep that would definitely be better.  changed and added a small note on use.

**Review by jeff-mccoy — APPROVED** [review]

**Pull request merged**